static/app/views/insights/common/queries/useReleases.tsx

```diff
@@ -13,14 +14,22 @@ import useApi from 'sentry/utils/useApi';
 import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import usePageFilters from 'sentry/utils/usePageFilters';
+import type {ReleasesSortByOption} from 'sentry/views/insights/common/components/releasesSort';
 
-export function useReleases(searchTerm?: string) {
+export function useReleases({
+  searchTerm,
+  sortBy,
+}: {
+  searchTerm?: string;
+  sortBy?: ReleasesSortByOption;
+} = {}) {
```

Comment:
```suggestion
export function useReleases({
  search,
  sort,
}: {
  search: string | undefined;
  sort: ReleasesSortByOption | undefined;
}) {
```

Fully optional APIs are not great, as people tend to always set the minimal amount of options and miss the rest (+ the usage starts to vary). Since it's not many options here, making these required will make it explicit when the search and sort are used vs when they are not. Minor nit on the param wording is that searchTerm could just be search, and sortBy to be concise